custom-vu/src/main/kotlin/jces1209/vu/JiraCloudScenario.kt

```diff
@@ -61,6 +59,11 @@ class JiraCloudScenario : Scenario {
                 jira = jira,
                 meter = meter,
                 projectMemory = similarities.projectMemory
+            ),
+            customizeColumns = CustomizeColumns(
```

Comment:
you can move it in ScenarioSimilarities, e.g. BrowseBoards

Reply:
for me it is easier to read code for such implementation. i'd leave as is